packages/api-db-mongodb/src/schema/index.ts

```diff
@@ -1 +1,4 @@
 export * from './0'
+export * from './1'
+export {DBUser} from './1'
+export {CollectionName} from './1'
```

Comment:
This doesn't look right to me. Was the idea to copy the whole schema `0` and then add the changes and make a new version or what was the plan?

Reply:
They way you've implemented it was the way I've planned. Because we don't have an explicit schema in MongoDB I thought having types for all the "schema" versions would be nice and help in migrating. But I can see how this would get convoluted quickly and there's no way to add new values into the `CollectionName` enum.